modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/schema/InternalIgniteTable.java

```diff
@@ -38,6 +39,13 @@
  * Ignite internal table.
  */
 public interface InternalIgniteTable extends IgniteTable {
+    /**
+     * Gets.
```

Comment:
I hope, the method really 'gets' smth. =)
What is the purpose if default 'null' value? I'm ok, if it is temporary solution.
```suggestion
     * Gets table id.
```

Reply:
Thanks, remove default imlementation

Reply:
Removed method because ID is avalable via 
```
InternalIgniteTable tbl; 
tbl.table().tableId();
```